lib/oauth-db.js

```diff
@@ -1,35 +1,55 @@
 var Boom = require('boom');
 var crypto = require('crypto');
 var Hoek = require('hoek');
+var pg = require('pg');
+var queries = require('./queries');
+var connString = process.env.POSTGRE_CONNECTION_STRING;
 
-var OAuthDB = function OAuthDB(clients, authorizations, tokens) {
-  this.clients = {};
-  this.authCodes = {};
-  this.accessTokens = {};
+var secret = process.env.SERVER_SECRET;
```

Comment:
Also, is there a recommendation for how much entropy this should have to be considered secure?